operator/example-cassdc-yaml/dse-6.8.x/example-cassdc-minimal.yaml

```diff
@@ -23,3 +23,7 @@ spec:
     jvm-server-options:
       initial_heap_size: "800M"
       max_heap_size: "800M"
+      additional-jvm-opts:
+        # As the database comes up for the first time, set system keyspaces to RF=3
+        - "-Ddse.system_distributed_replication_dc_names=dc1"
+        - "-Ddse.system_distributed_replication_per_dc=3"
```

Comment:
💯  I noticed after we spoke that this was missing on this example. Can you put it in the Cassandra minimal one too?